test/sys/test_ioctl.rs

```diff
@@ -1,7 +1,7 @@
 #![allow(dead_code)]
 
 // Simple tests to ensure macro generated fns compile
-ioctl!(do_bad with 0x1234);
+ioctl!(bad do_bad with 0x1234);
```

Comment:
We also need a test here that uses a comment with all of the various forms of the `ioctl!` macro given that this is the functionality you're really adding here.